_posts/2020-03-31-April-open-source-report.markdown

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: April-open-source-report
+date: 2020-03-31 20:12:12
+categories:
+author: bronzdoc
+
+---
+
+# Ombulabs Open Source report
+
+![img](https://external-preview.redd.it/zbCp6e3VikD9vsKOtTGVKI90UnWIOZN5NMRNSvgvXxE.jpg?width=960&crop=smart&auto=webp&s=9318c01ae704565ad90f4736aec5af6b671d98fa)
```

Comment:
@bronzdoc Could you add an alt text for this image? We want to be more accessible when showing images.